### PR TITLE
feat: move nextjs frontend in

### PR DESCRIPTION
**Pull request by Kimsy99**
## What is this about

Migrate frontend (nextjs, chakra, wagmi, and rainbowkit) from [here](https://github.com/Kimsy99/zk-form) to this repo.

### CODE REVIEW

File: ui/.github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: ci
```
Comment (cwkang1998) [inline ui/.github/workflows/ci.yaml:1]: Do you want to keep this ?

File: ui/LICENSE [hunk 2]
```diff
@@ -0,0 +1,24 @@
+This is free and unencumbered software released into the public domain.
```
Comment (cwkang1998) [inline ui/LICENSE:1]: Not important but I think we can remove this Unlicensed license.

File: ui/package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "rainplate",
+  "description": "A Versatile, Extensible Dapp Boilerplate built with Rainbowkit, Next.js, and Chakra-ui.",
+  "author": "asnared <https://github.com/abigger87>",
+  "private": false,
+  "license": "Unlicense",
```
Comment (cwkang1998) [inline ui/package.json:6]: If you want to maybe you can change this?

**Review by cwkang1998 — APPROVED** [review]
Honestly looks good to me, just some nit picks on non-important stuff, feel free to ignore those. Otherwise you can merge as you like.

**Comment by chee-chyuan** [comment]
can self merge guys no problem

**Comment by cwkang1998** [comment]
Merging this.

**Pull request merged**